build-monitor-plugin/src/main/resources/com/smartcodeltd/jenkinsci/plugins/buildmonitor/BuildMonitorView/configure-entries.jelly

```diff
@@ -56,6 +56,14 @@
     <f:textbox name="title" value="${it.title}"/>
   </f:entry>
 
+  <f:entry title="${%Include Branches}" help="${descriptor.getHelpFile('includeBranches')}">
+    <f:textbox name="include" field="include" value="${it.include()}"/>
```

Comment:
Could we call the property `includeBranches` or `branchesToInclude` so that the property provides a bit more of a context?